arctic-report-card/ARCRC.ttl

```diff
@@ -232,508 +262,647 @@ Find current and past assessments of the Arctic Report Card Vital Signs by visit
+                   dc:date "2024-07-30T19:43:23Z"^^xsd:dateTime ;
+                   rdfs:label "Western Alaska Salmon"@en ;
+                   skos:relatedMatch "http://purl.dataone.org/odo/SALMON_00000215" .
+
+
+###  http://purl.dataone.org/odo/ARCRC_00000018
+odo:ARCRC_00000018 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000040 ;
+                   obo:IAO_0000115 "Any product of the condensation of atmospheric water vapor that falls under gravity" ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-07-30T19:43:23Z"^^xsd:dateTime ;
+                   dc:description "This Key Variable is used to inform Vital Sign, 'Precipitation.'" ;
+                   rdfs:label "precipitation"@en ;
+                   owl:sameAs "http://purl.obolibrary.org/obo/ENVO_01000703" .
+
+
+###  http://purl.dataone.org/odo/ARCRC_00000019
+odo:ARCRC_00000019 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000040 ;
+                   obo:IAO_0000115 "Estimated total amount of ice in the sediment column and represents total ice content" ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-07-30T19:43:23Z"^^xsd:dateTime ;
+                   dc:description "This Key Variable is used to inform Other Indicator, 'Subsea Permafrost'" ;
+                   rdfs:label "total ice content"@en ;
+                   skos:relatedMatch "http://purl.dataone.org/odo/ECSO_00002173" .
+
+
+###  http://purl.dataone.org/odo/ARCRC_00000020
+odo:ARCRC_00000020 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000040 ;
+                   obo:IAO_0000115 "Long-term documentation of weather, ocean, sea ice, and landscape conditions and context to wildlife, harvesting, and cultural and community activities" ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-07-30T19:43:23Z"^^xsd:dateTime ;
+                   dc:description "This Key Variable is used to inform Frostbite, 'Working with Communities to Observe the Arctic'" ;
+                   rdfs:label "indigenous observations"@en .
+
+
+###  http://purl.dataone.org/odo/ARCRC_00000021
+odo:ARCRC_00000021 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000040 ;
+                   obo:IAO_0000115 "A measurement or estimate of the number of salmon, based on counts, densities, weights, or other methods." ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-07-30T19:43:23Z"^^xsd:dateTime ;
+                   dc:description "This Key Variable is used to inform Frostbite, 'Western Alaska Salmon'" ;
+                   rdfs:label "salmon abundance"@en ;
+                   owl:sameAs "http://purl.dataone.org/odo/SALMON_00000504" .
+
+
+###  http://purl.dataone.org/odo/ARCRC_00000022
+odo:ARCRC_00000022 rdf:type owl:Class ;
```

Comment:
Please relate this to the measuremnt types from the salmon ontolofy on run size and escapement

Reply:
Added equivalentClass to salmon escapement count [http://purl.dataone.org/odo/SALMON_00000479](http://purl.dataone.org/odo/SALMON_00000479)

Reply:
Is it equivalent? Or is escapement a subset? I'm not sure what the variable here represents, happy to discuss on slack.

Reply:
Updated SALMON ontology with "run size" term and added "run size" as an equivalentClass to "salmon run size"
`owl:equivalentClass "http://purl.dataone.org/odo/SALMON_00000573"`